builtInFunctions/esdtNFTSetNewURIs.go

```diff
@@ -0,0 +1,120 @@
+package builtInFunctions
```

Comment:
you missed esdtMetaDataUpdate - which is slightly different from recreate which does the following:  If nothing is received for a given attribute we keep the old version of that.

Reply:
Added.